docs/modules/mechanics/filters.mdx

```diff
@@ -1216,11 +1292,15 @@ This filter is a [Dynamic Filter](#dynamic-filters).
 
 ```
 <filters>
-    <!-- Pulses activation for 15 seconds in a 60 second time period -->
+    <!-- Pulses activation for 15 seconds in a 60 seconds time period -->
```

Comment:
Pretty sure the former was correct? "60 second time period" sounds more correct than with the s